docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,373 @@
+---
+title: Host Certificates with Machine ID
+description: Issuing Host Certificates with Machine ID for OpenSSH servers.
+---
+
+In earlier versions of Teleport, host certificates for OpenSSH servers were required to be created through `tctl`. While host certificates created through `tctl`
+are still a good option for some use cases, they must be manually rotated, and are not an ideal option where regular certificate rotation is a
+requirement for a strong security posture.
+
+Short-lived and regularly rotated host certificates provide a number of security improvements wherever applicable. Regular certificate rotation reduces risk
+by ensuring that any potentially stolen certificates are usable for a shorter period of time. Additionally, when coupled with Teleport's [RBAC](https://goteleport.com/docs/access-controls/guides/role-templates/)
+support and host certificate **Principals** and **Predicates**, limitations can be applied to both the process of creating host certificates, as well as the
+host certificates themselves.
+
+In this guide, you will learn how to create host certificates through Machine ID for OpenSSH servers, allowing for the benefits of host certificates with OpenSSH nodes,
+and reducing risk by ensuring that short-lived certificates adhering to the principle of least privelege can be successfully applied to Teleport clusters.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+(!/docs/pages/includes/tctl.mdx!)
+
+- OpenSSH server `sshd` version 6.9 or above installed. The SSH port on this host must be open to traffic from the Teleport Proxy Service host.
+
+  ```code
+  $ ssh -V
+  ```
+
+## Step 1/4. Download and install Teleport (=teleport.version=)
+
+In this step, you will be downloading and installing Teleport binaries onto a remote machine. For the purposes of this guide, this machine will be referred to as the `Remote Device` and exists separately
+from the Teleport cluster.
+
+Each Teleport package hosted on our downloads page ships with several useful
+binaries, including `teleport`, `tctl`, `tsh`, and `tbot`:
+
+- `teleport` is the daemon used to initialize a Teleport cluster; this binary is not used in this guide.
+- `tctl` is the administrative tool you will use to create the bot user.
+- `tsh` is the client tool you will use to log in to the OpenSSH server.
+- `tbot` is the Machine ID tool you will use to associate a bot user with a machine.
+
+Machine ID is available starting from the Teleport 9.0.0 release. Download the appropriate Teleport package for your platform from our [downloads page](https://goteleport.com/teleport/download).
```

Comment:
> Download the appropriate Teleport package for your platform from our [downloads page](https://goteleport.com/teleport/download).

Rather than this, what would you think of using our `install-linux.mdx` partial so users don't need to leave this guide?

Reply:
This makes sense to me. This section was built from our pre-existing Machine ID documentation, and I suspect that we may use it in our documentation elsewhere. I created an issue to follow up on this 

https://github.com/gravitational/teleport/issues/17306